src/main/webapp/app/faq/faq.component.ts

```
@@ -0,0 +1,123 @@
+    selector: 'jhi-faq',
+    templateUrl: './faq.component.html',
+    styleUrls: [],
+    standalone: true,
+    imports: [ArtemisSharedModule, CustomExerciseCategoryBadgeComponent, ArtemisSharedComponentModule, ArtemisMarkdownModule],
+})
+export class FaqComponent implements OnInit, OnDestroy {
+    faqs: Faq[];
+    filteredFaqs: Faq[];
+    existingCategories: FaqCategory[];
+    courseId: number;
+    hasCategories: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)